### DISCUSSION TITLE
Blurry HDR Environment Texture

### DISCUSSION THREAD

**Discussion by knightcube** [pmndrs/drei#675] [Q&A]
I used the example [here](https://github.com/pmndrs/drei#environment) to load an HDRI environment texture like this

```
 <Environment files="environment3.hdr" preset={null} background />
```

I used this environment - https://polyhaven.com/a/empty_warehouse_01.

It looks very blurry and not as sharp as shown in the preview (in the above URL). Almost every HDR file results in blurry or grainy output. You can even try it with the presets. Is there any way to make it look as good as the preview? 🤔

I did some more research on this and the same issue had been reported here - https://github.com/mrdoob/three.js/issues/197

This is what it looks like in my browser👇

![tNw1wXeK3P](https://user-images.githubusercontent.com/22871961/142978874-9dbfccf6-f2ac-4d2e-aaa5-3849b1e4f66b.jpg)

**Comment by ffdead** [comment]
Did you try downloading a higher res hdr image? The presets are only 1k

<img width="160" alt="Screenshot 2021-12-03 at 11 28 12" src="https://user-images.githubusercontent.com/420472/144587534-be13a810-9a18-4a94-8b75-c07fffdcc019.png">

**Reply by knightcube** [reply]
I tried with the 8K too.

**Comment by SimonDrastich** [comment]
How is this closed without resolution? I have same issue

**Comment by joshbedo** [comment]
Same issue the DREI HDRs look terrible on the ground
